libs/partners/anthropic/langchain_anthropic/chat_models.py

```diff
@@ -24,6 +25,30 @@
 _message_type_lookups = {"human": "user", "ai": "assistant"}
 
 
+def _format_image(image_url: str) -> Dict:
```

Comment:
FYI @dqbd the default anthropic format won't render properly in langsmith if implemented this way 

Think fine here